LanguageFeatures/Augmentation-libraries/type_inheritance_A01_t01_lib.dart

```diff
@@ -4,53 +4,53 @@
 
 /// @assertion An augmenting declaration may have no type annotations for a
 /// return type, variable type, parameter type, or type parameter bound type. In
-/// the last case, that includes omitting the extends keyword. For a variable or
-/// parameter, a var keyword may replace the type.
+/// the last case, that includes omitting the `extends` keyword. For a variable
+/// or parameter, a `var` keyword may replace the type.
 ///
 /// @description Check that augmenting declaration may have no type annotations
 /// for a return type.
 /// @author sgrekhov22@gmail.com
 
-// SharedOptions=--enable-experiment=macros
+// SharedOptions=--enable-experiment=augmentations,enhanced-parts
 
 part of 'type_inheritance_A01_t01.dart';
 
-augment get topLevelGetter => "Augmented top-level getter";
+augment get topLevelGetter;
 augment topLevelFunction() => "Augmented top-level function";
 
 augment class C {
-  augment static get staticGetter => "Augmented static getter";
+  augment static get staticGetter;
   augment static staticMethod() => "Augmented static method";
   augment get instanceGetter => "Augmented instance getter";
-  augment instanceMethod() => "Augmented instance method";
+  augment instanceMethod();
 }
 
 augment mixin M {
-  augment static get staticGetter => "Augmented static getter";
+  augment static get staticGetter;
   augment static staticMethod() => "Augmented static method";
   augment get instanceGetter => "Augmented instance getter";
-  augment instanceMethod() => "Augmented instance method";
+  augment instanceMethod();
 }
 
 augment enum E {
   augment e0;
```

Comment:
Wouldn't this be an error today?
https://github.com/dart-lang/language/blob/main/working/augmentation-libraries/feature-specification.md#augmenting-enums